src/test/java/com/google/research/bleth/simulator/TracingSimulationIT.java

```diff
@@ -0,0 +1,75 @@
+package com.google.research.bleth.simulator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TracingSimulationIT {
```

Comment:
I assume "IT" means something... Integration test maybe?
Prefer to spell this out fully; acronyms are confusing.

Reply:
By default, the Failsafe Plugin automatically includes all test classes which follow this naming convention:
https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html

(It can be manually configured if needed, but I guess following the 'IT' convention is ok).

Reply:
OK - I wasn't aware of this plugin (as usual, inside Google we have different tools).